xml/System.Runtime.Serialization/FormatterServices.xml

```diff
@@ -307,9 +307,11 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- Generally, the serializable members of a class are the non-transient, non-static members such as fields and properties. To be included, properties must have both a getter and a setter. A class that implements <xref:System.Runtime.Serialization.ISerializable> or has a serialization surrogate does not have to serialize all these members, or can serialize additional members.  
+ Generally, the serializable members of a class are non-transient, non-static members such as fields and properties. To be included, properties must have both a getter and a setter. A class that implements <xref:System.Runtime.Serialization.ISerializable> or has a serialization surrogate does not have to serialize all these members, or can serialize additional members.  
   
- Currently, you do not need to do anything different unless `context` has the <xref:System.Runtime.Serialization.StreamingContextStates> value <xref:System.Runtime.Serialization.StreamingContextStates.Clone> set.  
+ Currently, you do not need to do anything different unless `context` has the <xref:System.Runtime.Serialization.StreamingContextStates.Clone?displayProperty=nameWithType> value set.  
```

Comment:
not related to you changes but this sentence seems a bit lost. "you do not need to do anything different" for what?